src/inc/til/small_vector.h

```diff
@@ -278,6 +278,12 @@ namespace til
             return tmp;
         }
 
+        [[nodiscard]] friend constexpr small_vector_iterator operator+(const difference_type off, small_vector_iterator next) noexcept
+        {
+            next += off;
+            return next;
+        }

```

Comment:
This was missing. It's required for `til::small_vector` to fulfill the `Range` concept and be implicitly convertible to a `std::span`.